### ISSUE TITLE
Add sound

### ISSUE THREAD

**Issue by jbphet** [phetsims/friction#148]
During the 11/15/2018 status meeting it was decided that it's time to move forward with the implementation of the sound design for this simulation.

The [sound design prototype](https://phet-dev.colorado.edu/html/phet-io-wrapper-sonification/1.0.0-dev.86/phet-io-wrapper-sonification/friction/html/friction-sonification.html?sonificationFile=sonificationDefault&accessibility) should serve as the main specification.

**Comment by jbphet** [comment]
@emily-phet - Is there are design doc for this sound design?  Nothing is coming up for me on Google Drive if I search on "Friction Sound Design", and I don't recall seeing one.  I can easily work from the prototype, but thought I should make sure if that is the plan.

**Comment by jbphet** [comment]
Based on the discussion in #146, the "cooling" sound should be an enhanced sound and everything else should be a basic sound.

**Comment by emily-phet** [comment]
Yes - the latest prototype you've worked on with @Ashton-Morris is serving as the 'design doc'. Also, as you describe above - cooling sound is an 'enhanced' sound, all other sounds in latest prototype are the default/basic sound.

**Comment by jbphet** [comment]
One difference from the sound design prototype is that the pickup and drop sounds should be produced when using keyboard nav (this does not happen in the sound design wrapper prototype).

**Comment by zepumph** [comment]
While talking with @jbphet, we discussed that the above comment should not be implemented until https://github.com/phetsims/scenery-phet/issues/421 is finished. I'll ping this issue when we are done over there.

**Comment by jbphet** [comment]
The sound design is implemented.  I've published a dev version https://phet-dev.colorado.edu/html/friction/1.5.0-dev.24/phet/friction_en_phet.html.  Assigning to @Ashton-Morris and @emily-phet for what is essentially a final review of the sound design and implementation.  Note that mutli-platform testing will be done as part of the release testing, so the review should focus on the functionality.

**Comment by zepumph** [comment]
Adding blocks publication label, because this feels like it should be done before we make an RC tomorrow.

**Comment by emily-phet** [comment]
@jbphet I reviewed the sim and found some issues related to sound volume changes during use. I've started a separate issue for this. Otherwise, sounds great!

**Comment by emily-phet** [comment]
Sound design was finalized and implemented. Just in final QA testing stages now. Closing, as other issues have been/will be created as problems are found.
